server/questions.js

```diff
@@ -85,6 +85,13 @@ var prepareQuestionData = function(question, callback){
             questionToAdd.rightSide = question.rightSide;
             break;
 
+        case common.questionTypes.CHOOSEALL.value:
+			questionToAdd.type = common.questionTypes.CHOOSEALL.value;
```

Comment:
spaces fam. You killing me :(

Reply:
its github man

Reply:
@mehtadarshan1, @ahmedqarmout2 and I will help you fix your spacing tomorrow. It is not git's fault as the spacing for everyone else's PR is fine.

Reply:
![image](https://user-images.githubusercontent.com/24999877/32504189-3a0f1150-c3ad-11e7-848e-8d3188b0d7fd.png)

Reply:
this is 4 spaces and this is what you guys use as well, not sure what is different?